content/artifacts/artifact-automove/plugin.js

```diff
@@ -0,0 +1,387 @@
+// Automove artifacts to spacetime rip and withdraw
+
+import { 
+  PlanetType,
+  PlanetLevel,
+  BiomeNames
+} from "https://cdn.skypack.dev/@darkforest_eth/types";
+
+import {
+  coords,
+  canWithdraw
+} from 'https://plugins.zkga.me/utils/utils.js';
+
+// Execute 5 moves or withdraws every time
+const CONCURRENCY = 5;
+
+// Trigger automove every 3 minutes
+const AUTO_INTERVAL = 60 * 3;
+
+// debug mode
+const DEBUG = 0;
+
+const planetLevels = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
+
+
+function distance(from, to) {
+    let fromloc = from.location;
+    let toloc = to.location;
+    return Math.sqrt((fromloc.coords.x - toloc.coords.x) ** 2 + (fromloc.coords.y - toloc.coords.y) ** 2);
+}
+
+
+function isSpaceRip(planet) {
+  return planet.planetType === PlanetType.TRADING_POST;
+}
+
+
+function isActiveArtifact(fromPlanet, artifactId) {
+  let artifact = df.getActiveArtifact(fromPlanet);
+  if (artifact && artifact.id == artifactId) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+
+function isCooldown(artifact) {
+    if (artifact) {
+      if (artifact.lastDeactivated == 0) {
+        return false;
+      }
+      if (artifact.artifactType == 5) {
+        // Wormhole
+        if (Date.now() > (artifact.lastDeactivated + 48 * 60 * 60) * 1000) {
+          return false;
+        }
+      }
+      if (Date.now() > (artifact.lastDeactivated + 24 * 60 * 60) * 1000) {
```

Comment:
Can we refactor 24 * 60 * 60 as a constant listed at the top of the file please, so it can be easily updated?